fakebook/src/app/component/main-view/main-view.component.spec.ts

```diff
@@ -0,0 +1,66 @@
+import { ComponentFixture, fakeAsync, inject, TestBed } from '@angular/core/testing';
+import { By } from '@angular/platform-browser';
+import { NavigationBehaviorOptions , Router } from '@angular/router';
+import { NEVER } from 'rxjs';
+import { AuthService } from '../../service/auth.service';
+import { MainViewComponent } from './main-view.component';
+
+describe('MainViewComponent', () => {
+  let component: MainViewComponent;
+  let fixture: ComponentFixture<MainViewComponent>;
+  let navigateSpy: any;
+  let isAuth: boolean;
```

Comment:
this isn't right, is it? shouldn't you be setting and getting this property lower down?